src/spyglass/position/v1/position_dlc_pose_estimation.py

```diff
@@ -232,25 +232,38 @@ def make(self, key):
                 dlc_result.creation_time
             ).strftime("%Y-%m-%d %H:%M:%S")
 
-            logger.logger.info("getting raw position")
-            interval_list_name = (
-                convert_epoch_interval_name_to_position_interval_name(
-                    {
-                        "nwb_file_name": key["nwb_file_name"],
-                        "epoch": key["epoch"],
-                    },
-                    populate_missing=False,
+            # get video information
+            _, _, meters_per_pixel, video_time = get_video_path(key)
+            # check if a position interval exists for this epoch
+            try:
+                interval_list_name = (
+                    convert_epoch_interval_name_to_position_interval_name(
+                        {
+                            "nwb_file_name": key["nwb_file_name"],
+                            "epoch": key["epoch"],
+                        },
+                        populate_missing=False,
+                    )
                 )
-            )
-            spatial_series = (
-                RawPosition()
-                & {**key, "interval_list_name": interval_list_name}
-            ).fetch_nwb()[0]["raw_position"]
-            _, _, _, video_time = get_video_path(key)
-            pos_time = spatial_series.timestamps
-            # TODO: should get timestamps from VideoFile, but need the video_frame_ind from RawPosition,
-            # which also has timestamps
-            key["meters_per_pixel"] = spatial_series.conversion
+                raw_position = True
+            except KeyError:
+                raw_position = False
+
+            if raw_position:
+                logger.logger.info("Getting raw position")
+                spatial_series = (
+                    RawPosition()
+                    & {**key, "interval_list_name": interval_list_name}
+                ).fetch_nwb()[0]["raw_position"]
+                pos_time = spatial_series.timestamps
+                reference_frame = spatial_series.reference_frame
+                comments = spatial_series.comments
+            else:
+                pos_time = video_time
+                reference_frame = ""
+                comments = "no comments"
```

Comment:
A similar thing could be done here as above with `getattr` and supplying these as defaults